common-files/utils/sync-files.mjs

```diff
@@ -0,0 +1,145 @@
+/* ignore unused exports */
+import axios from 'axios';
+import fs from 'fs';
+import config from 'config';
+import logger from './logger.mjs';
+import downloadFile from './httputils.mjs';
+
+const {
+  CONTRACT_ARTIFACTS,
+  DEPLOYMENT_FILES_URL: { CONTRACT_FILES_URL = '' },
+  DEPLOYMENT_FILES_URL: { CIRCUIT_FILES_URL = '' },
+} = config;
+
+export async function checkContractsABI() {
+  if (CONTRACT_FILES_URL !== '') {
```

Comment:
```
if (CONTRACT_FILES_URL) { ... }
```

That should be enough. Empty string in JS is a falsy value.

Reply:
Done, thank you